docs/index.html

```diff
@@ -154,7 +154,7 @@ <h6>Paste that at a Terminal prompt.</h6>
                                 <p>For detailed installation instruction go to: </br><a href="https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html#installation">https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html#installation</a></p>
                                 <p>Quick Start: <a href="https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html">https://trinity-client.readthedocs.io/en/latest/guides/quickstart.html</a></p>
                                 <p>Github: <a href="https://github.com/ethereum/trinity">https://github.com/ethereum/trinity</a></p>
-                                <p>Serenity (Eth 2.0) Beacon Chain implementation: <a href="https://github.com/ethereum/trinity/blob/master/eth2/README.md">https://github.com/ethereum/trinity/blob/master/eth2/README.md</a></p>
+                                <p>[Eth2.0 Beacon Chain Implementation](./eth2/README.md)</p>
```

Comment:
I'm sorry, my previous comment was totally wrong.  I though I was looking at the README for the trinity repo.  Your previous format was correct using the HTML link.